Tasks/CargoAuthenticateV0/Tests/L0.ts

```diff
@@ -0,0 +1,16 @@
+import fs = require('fs');
+import assert = require('assert');
+import path = require('path');
+
+describe('CargoAuthenticateV0 Suite', function () {
+    before(() => {
+    });
+
+    after(() => {
+    });
+
+    it('Does a basic hello world test', function(done: MochaDone) {
+        // TODO - add real tests
```

Comment:
Should we have tests?

Reply:
I will try to add some later on, we need to add more tests for most of the protocols we support